### ISSUE TITLE
why just use 1 in cgt[cgt == 255] = 1 in dfn dataloader？

### ISSUE THREAD

**Issue by yaodonggggggg** [ycszen/TorchSeg#35]
Is it more robust to other numbers?

**Comment by my462** [comment]
I am just goona pop up this issue. Why did you close it?

**Comment by yaodonggggggg** [comment]
> I am just goona pop up this issue. Why did you close it?

Sorry to reply late.I forget why i closed this,maybe it's a mistake.

**Comment by my462** [comment]
Well, in my opinion, it's wrong to give the edge pixels wrong label 1 which is class aero.

**Comment by yaodonggggggg** [comment]
> Well, in my opinion, it's wrong to give the edge pixels wrong label 1 which is class aero.

It is not label,just a value of gray.You can test the image transform by a little part code.If you delete cgt[cgt == 255] = 1,the egde is clear to see.So i think maybe the 1 is a part of better learning.

**Comment by my462** [comment]
Got you. I missunderstand the canny function of cv2.That's why I assume in cgt the label info still remains while actually canny will only leave 0 and 255. And by turning 255 to 1 should help training.
